### PR TITLE
Bookstore: styling

### PR DESCRIPTION
**Pull request by ivaan1992**
For this project I:

- Added style.
- Added a screenshot of the project to the README.

### CODE REVIEW

**Review by er-contreras — CHANGES_REQUESTED** [review]
Ignore this...

**Comment by eri8-9** [comment]
> Hi @ivaan1992 ,
> 
> Good job so far! There are some issues that you still need to work on to go to the next project but you are almost there!
> 
> almost there
> 
> ## Required Changes ♻️
> _Check the comments under the review._
> 
> ## Optional suggestions
> _Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._
> 
> Cheers and Happy coding!👏👏👏
> 
> Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
> 
> **_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**
> 
> _As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

#  *Please ignore this

File: src/App.css [hunk 1]
```diff
@@ -1,31 +1,93 @@
...
+header {
+  border: 3px solid #e8e8e8;
   display: flex;
-  margin-right: 20px;
-  justify-content: end;
-  align-items: center;
+  flex-direction: row;
+  height: 10vh;
+}
+
```
Comment (er-contreras) [inline src/App.css:32]: - Please stick to the design of the Bookstore

Your design:
Screen Shot 2022-01-29 at 16 33 10

The design we want to achieve:
Screen Shot 2022-01-29 at 16 33 41

 * Stick to the design as much as possible

**Review by er-contreras — CHANGES_REQUESTED** [review]
Hi @ivaan1992 , 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

<FEEL FREE TO CHANGE THIS MESSAGE TO YOUR PERSONAL WORDS>

almost there

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: src/App.css [hunk 2]
```diff
@@ -1,31 +1,235 @@
...
+.lesson {
+  font-family: 'Roboto Slab', serif;
+  font-size: 1rem;
+  font-weight: 300;
+  margin-bottom: 40px;
+}
+
+#bookInput {
+  width: 40%;
+}
```
Comment (rukundoeric) [inline src/App.css]: - [x] <b>REQUIRED:</b> There are still few things we need to fix to make our app look more like the provided design. Kindly review the following issues and make sure to fix them.

| Original      | Yours |
  | ----------- | ----------- |
| 80047697-0c3f6280-8506-11ea-9416-a4f1800e2329 | Screen Shot 2021-08-26 at 00 15 40 |

  - In the mockup design, the app is more like a card in the center of the screen.
  - You still have many margins gaps that aren't in the mockup design. Kindly address them.
  - Remember to make sure that left and right margins are the same. in other words, The card must be in the center of the horizontal orientation.

**Review by rukundoeric — CHANGES_REQUESTED** [review]
Hi @ivaan1992, 

_Great job so far 👏 💯 👍. You've done well implementing some of the requested changes. However, there are still some which aren't addressed yet. Please, check the comments below._
## Highlights
  - Follows git workflow.  💯
  - Codebase looks clean and professional. 💯 
  - Addressed some of the issues raised from the previous review. 💯 

## Required Changes ♻️
  - [x] Check the comments under the review.

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._
  
  - N/A 

Cheers and Happy coding!👏👏👏

__Feel free to leave any questions or comments in the PR thread if something is not 100% clear.__

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
> As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s).

**Review by arslanbisharat — APPROVED** [review]
# Approved 👏 

Hi there, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

To Highlight: 
- The Pull Request has a proper title and description. ✔️ 
- You have added a descriptive Readme file. Good Job. ✔️ 
- There is no linter error is present. ✔️ 
- Project has met all the technical aspects you needed for this project. Keep rocking! ✔️ 

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of code reviews per project which you can see from your dashboard. If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)_

**Pull request merged**